terraform/tags.tf

```diff
@@ -0,0 +1,11 @@
+module "beacons_label" {
+  source      = "git::https://github.com/cloudposse/terraform-null-label.git?ref=master"
+  namespace   = "mca"
+  environment = terraform.workspace
+  name        = "beacons"
+
+  tags = {
+    Owner           = "BeaconRegistry@mcga.onmicrosoft.com"
```

Comment:
Is that a valid mca email that should be used/confirmed by Paul?

Reply:
Yep, got it from Aniko.

Reply:
But not confirmed with @paulgutman -- Paul is this email OK?

Reply:
Yes I can see that in our GAL